app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -162,17 +170,115 @@
   <!-- footer -->
   <footer>
     <h4 style="padding-top: 60px;">Notes</h4>
-    <p>Data is for all activity at <%= @current_facility.name %>. If data appears out of date, tap the "Sync
-      pending" link on the home screen.</p>
-    <p>A follow-up patient is only counted for one visit each month. If a patient follows up multiple times in a month,
-      he or she is
-      only counted once.</p>
+    <p>Data is for all activity at <%= @current_facility.name %>. If data appears out of date, tap the "Sync pending" link on the home screen.</p>
     <h4 style="padding-top: 60px;">Thank you</h4>
-    <p>Thank you for all of your hard work treating patients with hypertension. Your work saves lives from heart
-      attacks
-      and strokes.</p>
+    <p>Thank you for all of your hard work treating patients with hypertension. Your work saves lives from heart attacks and strokes.</p>
     <div style="height: 100px;"></div>
   </footer>
+  </div>
+</div>
+
+<div id="progress-help-registered" class="progress-body" style="display: none;">
```

Comment:
The three `#progress-help-*` cards here and below are the extra explanations that have been added to the progress tab.